### PR TITLE
fix: Fixed some bugs in styling

### PR DESCRIPTION
**Pull request by accodes21**
Changed border-radius of profile pfp.
Added hover effect on profile card

### CODE REVIEW

File: data/users/accodes21.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Aarya Chopkar",
```
Comment (codewithvoid) [inline data/users/accodes21.json:2]: i think branch is outdated..this file shouldn't be updated

**Comment by codewithvoid** [comment]
Thanks for the fix @accodes21 :)

**Comment by accodes21** [comment]
Just updated the branch

**Comment by accodes21** [comment]
Updated the branch, now its ready to be reviewd @codewithvoid

**Review by codewithvoid — APPROVED** [review]

**Pull request merged**